src/app/services/website.service.ts

```diff
@@ -62,17 +62,17 @@ export class WebsiteService extends CRUDListService<ApiWebsiteWithCompany, ApiWe
   }
 
   readonly listUnregistered = (q?: string, start?: Date, end?: Date): Observable<HostWithReportCount[]> => {
-    return this.utils.getSecuredReportApiSdk.pipe(
+    return of(_ => _.next()).pipe(
```

Comment:
à quoi sert of(_ => _.next()) ? équivalent à empty() ?

Reply:
J'ai besoin de ça pour inclure l'initialisation des flags.
Si je par exemple j'écris

```
// service.ts
const fetch = () => {
  this.loading = true;
  return this.client.fetch().pipe(...)
};
```

`this.loading = true` va être trigger sans que je subscribe, donc sans que l'appel http soit fait.